BitmovinConvivaAnalytics/Classes/ConvivaAnalytics.swift

```diff
@@ -510,39 +534,48 @@ extension ConvivaAnalytics: BitmovinPlayerListenerDelegate {
 
     // MARK: - Ad events
     func onAdStarted(_ event: AdStartedEvent) {
+        var adInfo = event.ad.adInfo
         let adPosition: ConvivaSDK.AdPosition = AdEventUtil.parseAdPosition(
             event: event,
             contentDuration: player.duration
         )
-        var adAttributes = [String: Any]()
-        adAttributes["c3.ad.position"] = adPosition.rawValue
-        videoAnalytics.reportAdBreakStarted(
-            AdPlayer.ADPLAYER_CONTENT,
-            adType: AdTechnology.CLIENT_SIDE,
-            adBreakInfo: adAttributes
+        adInfo["c3.ad.position"] = adPosition.rawValue
+
+        adAnalytics.reportAdLoaded(adInfo)
```

Comment:
On iOS we do have the `AdManifestLoaded` event 🤔 Could this be considered for `reportAdLoaded`?

Reply:
The same exists on Android, but this is about the media file/stream itself.